### PR TITLE
Update Readme.Update Openshift versions in annotations

### PR DESCRIPTION
**Pull request by rensyct**
# Description
This PR 
- Updates the install steps of Operator in Readme.
- Updates the Openshift versions in annotations

# GitHub Issues
List the GitHub issues impacted by this PR:

| GitHub Issue # |
| -------------- |
| https://github.com/dell/csm/issues/743 |

# Checklist:

- [x] I have performed a self-review of my own code to ensure there are no formatting, vetting, linting, or security issues
- [ ] I have verified that new and existing unit tests pass locally with my changes
- [x] I have not allowed coverage numbers to degenerate
- [x] I have maintained at least 90% code coverage
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] I have maintained backward compatibility

# How Has This Been Tested?
Please describe the tests that you ran to verify your changes. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

### CODE REVIEW

**Review by panigs7 — APPROVED** [review]

**Review by rajendraindukuri — DISMISSED** [review]

File: README.md [hunk 1]
```diff
@@ -61,7 +61,32 @@ Dell Container Storage Modules Operator has been tested and qualified with
...
+## Installation of Operators (Both OLM and Non OLM) in Dev Mode
+  1. Clone the repo: `git clone https://github.com/dell/csm-operator.git` 
+  2. Navigate one level inside the cloned repo: `cd csm-operator`
+  3. Execute `make install` to install the CRDs
+  4. Execute `make run` to have the operator running.
+  5. Install any of the operands (CSI Driver) using another session 
+
+NOTE: Closing the session where operator is running will stop the operator.
+
+## Installation of Operators using Scripts
```
Comment (bharathsreekanth) [inline README.md:74]: Does S have to be capitalized in scripts?

**Review by bharathsreekanth — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (donatwork) [reply README.md:74]: Seems like if you capitalize Scripts then you should also capitalize the u in using. Thats the normal convention for headlines. Unless you chose to capitalize only the first letter of the phrase.

**Review by donatwork — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (rensyct) [reply README.md]: Corrected

File: README.md [hunk 2]
```diff
@@ -61,7 +61,32 @@ Dell Container Storage Modules Operator has been tested and qualified with
...
+## Installation of Operators (both OLM and Non OLM) in dev mode
+  1. Clone the repo: `git clone https://github.com/dell/csm-operator.git` 
+  2. Navigate one level inside the cloned repo: `cd csm-operator`
+  3. Execute `make install` to install the CRDs
+  4. Execute `make run` to have the operator running.
+  5. Install any of the operands (CSI Driver) using another session 
+
+NOTE: Closing the session where operator is running will stop the operator.
+
+## Installation of Operators using scripts
```
Comment (bharathsreekanth) [inline README.md:74]: Same as above.

**Review by bharathsreekanth — APPROVED** [review]
Minor comments

File: README.md [hunk 3]
```diff
@@ -61,7 +61,32 @@ Dell Container Storage Modules Operator has been tested and qualified with
     * OpenShift Clusters 4.11, 4.12 with RHEL 8.x & RHCOS worker nodes
 
 ## Installation
-To install Dell Container Storage Modules Operator please refer the steps given here at [https://dell.github.io/csm-docs/docs/deployment/csmoperator/](https://dell.github.io/csm-docs/docs/deployment/csmoperator/)
+
+## Installation of Operators (both OLM and Non OLM) in dev mode
```
[the remark this answers is not in the document]
Comment (rensyct) [reply README.md]: Corrected

**Review by donatwork — APPROVED** [review]

**Review by karthikk92 — APPROVED** [review]

**Pull request merged**